brainio-test/MANIFEST.in

```diff
@@ -1 +1,2 @@
 include brainio_test/lookup.csv
```

Comment:
this file seems to be deleted in this PR?

Reply:
Deleted?  Which file, the manifest or the lookup?

Reply:
oh sorry github rendered this weirdly, it's just some lines deleted in that file